spybot/recorder/client.py

```diff
@@ -83,14 +80,16 @@ def __client_end_session(self, ts_user: TSUser, reason_id: int):
         # old_activity = newest TSUserActivity for client_id (in channel_id)
         # insert endTime, reason_id into old_activity
         try:
-            old_activity = TSUserActivity.objects.order_by('-start_time').filter(tsuser=ts_user)[0]
+            old_activity = TSUserActivity.objects.order_by('-start_time').filter(tsuser=ts_user, end_time=None)
```

Comment:
maybe rename variable to represent multiple activities (i.e. plural)

Reply:
I did!